src/AmpWpPlugin.php

```diff
@@ -106,7 +110,20 @@ protected function get_bindings() {
 	 *                      to argument values.
 	 */
 	protected function get_arguments() {
-		return [];
+		return [
+			ServerTiming::class => [
+				// Wrapped in a closure so it is lazily evaluated. Otherwise,
+				// is_user_logged_in() breaks because it's used too early.
+				'verbose' => static function () {
+					return is_user_logged_in()
```

Comment:
I think this should also be checking if `current_user_can('manage_options')`. Non-admins shouldn't necessarily be able to get this information.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5061